client/styles/index.scss

```diff
@@ -19,24 +19,36 @@
     }
 }
 
-.AssignmentItem {
-    .sd-list-item__time__schedule {
-        color: $sd-blueDark !important;
-        font-weight: 500;
-        vertical-align: text-top;
-        padding-left: 0;
-    }
+.assignment--due-date {
+  padding: 0;
 
-    .label-icon--warning {
-        .sd-list-item__time__schedule {
-            color: $orange !important;
-        }
+  > * {
+    margin: 0!important;
+  }
+
+  > * + * {
+    margin-left: 6px!important;
+  }
 
-        .label {
-            margin-left: 0.75rem;
-        }
+  .sd-list-item__time__schedule {
+    color: $sd-blueDark !important;
+    font-weight: 500;
+    vertical-align: text-top;
+    padding-left: 0!important;
+  }
+  
+  &.label-icon--warning {
+    .sd-list-item__time__schedule {
+        color: $orange !important;
+    }
+  
+    .label {
+        margin-left: 0.75rem;
```

Comment:
To stay in line with the UI framework's Layout Principles, this should be `margin-left: 0.8rem;`